rtime.go

```diff
@@ -45,69 +43,58 @@ func Now() time.Time {
 				tm, err := time.Parse(time.RFC1123, resp.Header.Get("Date"))
 				resp.Body.Close()
 				if err == nil {
-					mu.Lock()
-					res = append(res, tm)
-					cond.Broadcast()
-					mu.Unlock()
+					results <- tm
```

Comment:
This will block forever unless there's the channel is buffered.

Reply:
I suppose if the select loop isn't started by the time the first result returns from a http call it could block, but I couldn't get that to happen when I ran the code. 

Maybe it is safer to use the buffered channel. Out of curiosity, did you run it and observe it blocking? If so, how'd you test?

Reply:
I added a `select{}` to the bottom of the test and `defer println(site)` to the go routine. You'll see only three of the seven make it past the channel. The other four just hang waiting for the channel receiver.

```go
go func(site string) {
	defer println(site)
	// ...	
}(site)
```

```go
func TestTime(t *testing.T) {
	// ...
	select {}
}
```

Reply:
You're right -- I guess my assumption was that it didn't matter if we ever receive the result from anything beyond the first 3 responses, but it's better to not block regardless. updated w/ the buffered chan